README.md

```diff
@@ -71,6 +71,45 @@ Then the following data is inserted in the store:
         as:deleted "thetimeofdelete"^^xsd:dateTime  .
 ```
 
+## Extending Forms
+
+### Predicates and Types
+
+Use the same model as regular forms, with the following differences:
+-   don't use the `sh:group` predicate to denote connections to the owning form (or its sections/listings). Instead use `ext:extendsGroup`
+-   don't use the `form:includes` predicate to denote that a field is part of a form. Instead use `ext:extendsForm`
```

Comment:
Is it just in this stage we want to use these predicates, or is it meant to be for a longer to. If it is the first sure, if it is the latter, I would think about formalising this and not use the ext prefix.

Reply:
very correct comment. We want it in the longer run but internal to our own service (for now). While this is still up for discussion, we still need to be able to work so i wrote it with and ext prefix in the spec until we are fine sharing this more broadly